### ISSUE TITLE
User session uptime

### ISSUE THREAD

**Issue by max-xface** [criteo/ocserv-exporter#3]
Can you add an output of how long the user session has been active?

**Comment by fdomain** [comment]
Yes, this is something doable.

Output example from `occtl --json show users`:
```
    "Connected at":  "2023-09-22 04:28",
    "_Connected at":  " 9h:46m",
```

What should be done is to parse the duration of "_Connected at" value.
Don't hesitate to propose a PR if you're interested, otherwise I'll have a look soon.

**Comment by fdomain** [comment]
After looking closer into this, the values returned in `Connected at` or `_Connected at` are not consistent enough (the format changes depending on the connection time).
I proposed [a PR](https://gitlab.com/openconnect/ocserv/-/merge_requests/374) on ocserv project to display the raw value of the connection start time (unix timestamp).

Depending on its acceptance, I'll make the required change on ocserv-exporter.
The only thing you'll have to do on Prometheus is to compute the uptime by doing something like:

`time() - vpn_user_start_time_seconds`

**Comment by fdomain** [comment]
The PR is accepted, so I added `vpn_user_start_time_seconds` metric (see https://github.com/criteo/ocserv-exporter/commit/11796a5e743c0d57556ebbc2269cdccb2e8860bd).

It relies on the presence of the `raw_connected_at` field in JSON output of occtl, which will be most likely available with version 1.2.3 of ocserv.
